File: area.py
#!/usr/bin/env python3
""""
Created by: Nina Mathew
Created on: April 17, 2023
This is the " value of one number raised to the power of another. " module
"""


def calculate_area(base_f, height_f) -> None:
    """The calculate_area() function calculates area of a rectangle, returns None."""

    # process
    area = base_f * height_f * 1 / 2

    # output
    print(f"The area is {area:.4f} cm²")
    print("")


def main() -> None:
    """The main() function just calls other functions, returns None."""

    base_str = input("Enter the base of the triangle(cm): ")
    height_str = input("Enter the height of a triangle (cm): ")
    try:
        base_f = float(base_str)
        height_f = float(height_str)
        if base_f <= 0 or height_f <= 0:
            print("Please enter a positive number.")
        else:
            calculate_area(base_f, height_f)
    except ValueError:
        print("Invalid input.")

    print("\nDone.")


if __name__ == "__main__":
    main()

File: Area.c
// Copyright (c) 2022 Nina Mathew All rights reserved
//
// Created by: Nina Mathew
// Created on: April 14, 2023
// This program does a for loop

#include <stdio.h>

void calculateArea() {
    float base;
    int ScanError = 0;
    float height;
    float area;
    // input
    printf("Enter the base of a triangle (cm): ");
    ScanError = scanf("%f", &base);
    printf("Enter the height of a triangle (cm): ");
    ScanError = scanf("%f", &height);

    if (ScanError == 0) {
        printf("Invalid input.\n");
    } else if (base < 0 || height < 0) {
        printf("Please Enter a Positive Number.\n");
    } else {
    // process
    area = base * height * 1/2;
    // output
    printf("The area is %.4f cm²\n\n", area);
}
}
int main() {
    // Call functions
    calculateArea();
    printf("\nDone.\n");
    return 0;
}
